### PR TITLE
Adding July meeting.

### PR DESCRIPTION
**Pull request by flagxor**

### CODE REVIEW

File: 2017/CG-05.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ Please register before the event [here](https://goo.gl/forms/LQVhuRuI22lK8Umw1).
 
 | Dates                    | Location          | Host       |
 |--------------------------|-------------------|------------|
-| 2017-07-17 to 2017-07-19 | San Francisco, CA | Google     |
+| 2017-07-17 to 2017-07-19 | Kirkland,WA       | Google     |
```
Comment (jfbastien) [inline 2017/CG-05.md]: Missing a space after the comma.

File: 2017/CG-07.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+* Technical presentation requirements (adapters, google hangouts/other accounts required, etc.)
+  - Presentations will be done with a Google Hangout.
+  - Contact host if you need alternatives.
+
+* Any other logistics required to participate in the meeting
+  TBD
+
+### Hotels
+
+[Nearby Hotels](https://www.google.com/maps/search/Hotels+near+google+kirkland/@47.6682914,-122.1933552,14z)
```
Comment (jfbastien) [inline 2017/CG-07.md]: Is there a preferred choice in the area?

File: 2017/CG-07.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
+![WebAssembly logo](/images/WebAssembly.png)
+
+## Agenda for the May meeting of WebAssembly's Community Group
+
+- **Host**: Google, Kirkland, WA
+- **Dates**: Monday-Wednesday May 17-19, 2017
```
Comment (stoklund) [inline 2017/CG-07.md]: *July*

File: 2017/CG-05.md (hunk 1 above)
Comment (adanilo) [inline 2017/CG-05.md]: There was consensus to change this to 18th through 20th July, can someone confirm?

Comment (jfbastien) [inline 2017/CG-05.md]: Brad wasn’t sure he could get the room for those dates. Consensus was that he should try to.

Comment (flagxor) [inline 2017/CG-05.md]: Just got them today.
Revising.

File: 2017/CG-07.md (hunk 3 above)
Comment (flagxor) [inline 2017/CG-07.md]: done

File: 2017/CG-07.md (hunk 2 above)
Comment (flagxor) [inline 2017/CG-07.md]: Added one.

File: 2017/CG-05.md (hunk 1 above)
Comment (jfbastien) [inline 2017/CG-05.md]: Still missing a space after the comma.

File: 2017/CG-07.md (hunk 3 above)
Comment (jfbastien) [inline 2017/CG-07.md]: I think you haven't pushed your update?

**Comment by flagxor** [comment]
That was really odd, it showed the changes on the branch, but didn't update the pull request.
Did a whitespace change and pushed and now it shows.

PTAL

**Pull request merged**